scripts/checkpoint_converters/convert_falcon_nemo_to_hf.py

```diff
@@ -94,6 +97,7 @@ def convert(input_nemo_file, output_hf_file, precision=None, cpu_only=False) ->
         model_config = MegatronGPTModel.restore_from(input_nemo_file, trainer=dummy_trainer, return_config=True)
         model_config.use_cpu_initialization = True
         model_config.tensor_model_parallel_size = 1
+        model_config.name = "te_gpt"
```

Comment:
Falcon should use `megatron_falcon_gpt` (or no need to set it here)

Reply:
Otherwise this PR looks good to me!

Reply:
I removed it from Falcon converter file. If this looks good to you know, can you please approve and merge?